Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_IOCs_-_Disable_expired_IOCs_in_XDR.yml

```diff
@@ -0,0 +1,908 @@
+    type: condition
+    task:
+      id: 31c74697-6736-411c-8b37-00c6584e6d53
+      version: -1
+      name: Has MD5
+      description: 'Are there MD5 indicators to be pushed '
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "33"
+      "yes":
+      - "31"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: iocs.indicator_type
+            iscontext: true
+          right:
+            value:
+              simple: File
+      - - operator: isExists
+          left:
+            value:
+              simple: iocs.md5
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 30,
+          "y": 1180
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "33":
+    id: "33"
+    taskid: 93b6ee78-4f0e-4c15-8f01-8376804ab43d
+    type: title
+    task:
+      id: 93b6ee78-4f0e-4c15-8f01-8376804ab43d
+      version: -1
+      name: Hashes Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "36"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -180,
+          "y": 1750
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: 21143669-630d-4a03-8e93-69d6c7eb1fc6
+    type: regular
+    task:
+      id: 21143669-630d-4a03-8e93-69d6c7eb1fc6
+      version: -1
+      name: xdr-iocs-disable
+      description: Disables IOCs in the XDR server.
+      script: Cortex XDR - IOC|||xdr-iocs-disable
+      type: regular
+      iscommand: true
+      brand: Cortex XDR - IOC
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      execution-timeout:
+        simple: "800"
+      indicator:
+        complex:
+          root: vetted_iocs
+          transformers:
+          - operator: join
+            args:
+              separator:
+                value:
+                  simple: ','
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 510,
+          "y": 1900
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {},
+    "paper": {
+      "dimensions": {
+        "height": 2555,
+        "width": 2590,
+        "x": -440,
+        "y": 20
+      }
+    }
+  }
+inputs:
```

Comment:
Let's add default values for those inputs. 
Default batch size, and the query to get the expired indicators

Reply:
There are default values in the main playbook, do you wat it here as well?